docs/distributions/debian/installation.md

```diff
@@ -1,93 +1,97 @@
+    Due to a bug in the Pop!_OS installer, the partition sizes shown by it are incorrect when using manual partitioning. As a workaround you may follow the instructions given in this [GitHub issue](https://github.com/elementary/installer/issues/620#issuecomment-1356978490) in the live ISO environment to fix the installer and then start the installation.
 
 1. Follow the [Pre-installation](https://wiki.t2linux.org/guides/preinstall) steps to prepare your Mac for installation.
-2. Boot into the Live ISO. You should now be in the GRUB boot menu. Select the option which is relevent to you.
-3. Start the installer and install it like normal until you get an option to manually specify partitions.
-4. Find the partition you made for Linux when you were following the Pre-installation steps. MAKE SURE TO SELECT THE RIGHT PARTITION OR ELSE YOUR DATA WILL BE LOST. Delete it to make free space. You'll need to make these partitions:
+2. Boot into the Live ISO. In the GRUB boot menu, select the option relevant to you.
+3. Start the installer and go through it like normal until you get to partitioning. Select the relevant option for **manual partitioning**.
+4. **WARNING: MAKE SURE TO SELECT THE RIGHT PARTITION OR YOU MAY LOSE YOUR DATA.** Find the partition you made for Linux during the Pre-installation step and delete it to make free space. You'll need to make these partitions:
 
-    1. If you want, you can make seperate partitions for **swap**, `/home`, `/boot` etc as you do in a normal PC.
+    1. You **must** create a **`/`** ("root") partition formatted as **ext4** or **btrfs**.
 
-    2. The partition to be made compulsorily is the one mounted at `/` and formatted to **ext4** or **btrfs**.
+    2. You may opt to make separate partitions for **`/home`**, **`/boot`**, **swap** etc. as you would normally.
 
-    3. If the installer used by your distro is `ubiquity`, which is the one used in Ubuntu, Linux Mint etc., then you can leave EFI Boot alone. If you are using a [separate EFI partition](https://wiki.t2linux.org/guides/windows/#using-seperate-efi-partitions), then you shall have to separate it out after installation by following [this guide](https://wiki.t2linux.org/guides/windows/#seperate-the-efi-partition-after-linux-is-installed).
+    3. If `ubiquity` is the installer used by your distro (used in *Ubuntu*, *Linux Mint* and others), then you can leave EFI Boot alone. If you are using a [separate EFI partition](https://wiki.t2linux.org/guides/windows/#using-seperate-efi-partitions), then you shall have to separate it out after installation by following [this guide](https://wiki.t2linux.org/guides/windows/#seperate-the-efi-partition-after-linux-is-installed).
```

Comment:
And other similar distros shouldn't be better?

Reply:
I'm not sure what you mean here.

Reply:
used in *Ubuntu*, *Linux Mint* and other similar distros

Reply:
Noted.